downstream/modules/automation-hub/proc-using-content-signing-services-in-pah.adoc

```diff
@@ -0,0 +1,35 @@
+[id="proc-using-content-signing-services-in-pah"]
+
+= Using content signing services in private automation hub
+
+After you have configured content signing on your {PrivateHubName}, you can manually sign a new collection or replace an existing signature with a new one so that users who want to download a specific collection have the assurance that the collection is intended for them and has not been modified after certification.
+
+Content signing on {PrivateHubName} provides solutions for the following scenarios:
+
+* Your system does not have automatic signing configured and you must use a manual signing process to sign collections.
+* The current signatures on the automatically configured collections are corrupted and must be replaced with new signatures.
+* Additional signatures are required for previously signed content.
+* You want to rotate signatures on your collections.
+
+.Procedure
+
+. In the `automationhub-terminal`, run the following command to publish your collections:
++
+----
+ansible-galaxy collection publish <ansible-test_collection-1.0.0.tar.gz> -c
+----
++
+[NOTE]
+====
+The `-c` option is used to ignore SSL certificate errors.
+====
```

Comment:
Removing the note also, which related to the first step and code that preceded it.